includes/RestApi/Themes/ThemeVariationsController.php

```diff
@@ -43,16 +44,32 @@ public function register_routes() {
 					'callback' => array( $this, 'get_theme_variations' ),
 					'permission_callback' => array( Permissions::class, 'rest_is_authorized_admin' ),
 				),
+				array(
+					'methods'  => \WP_REST_Server::EDITABLE,
+					'callback' => array($this, 'set_theme_variation'),
+					'permission_callback' => array(Permissions::class, 'rest_is_authorized_admin'),
+				),
 			)
 		);
 	}
 
 	/**
 	 * Retrieves the active themes variations.
 	 *
-	 * @return \WP_REST_Response|\WP_Error
+	 * @return \array|\WP_Error
 	 */
 	public function get_theme_variations( \WP_REST_Request $request ) {
+
+		$default = $request->get_param('defaultValue');
```

Comment:
Can you please explain the purpose of `defaultValue`?

Reply:
So basically some pages do need the 6 variations like the Theme Styles page even though the user navigates back, so if DefaultValue is true the function behaves as it used to behave earlier before this story, else it checks if the option exists, and handles accordingly.